### ISSUE TITLE
Misc UI fixes

### ISSUE THREAD

**Issue by charlesh88** [nasa/openmct#4843]
Periodic need to fix visual layout issues and improve the UI after user observations.

- [ ] Make Edit mode more apparent. People not realizing that they're in Edit mode - need to make more visually apparent, increase button function clarity.
- [ ] Macos: Scrollbars are not showing in Firefox, even in overflow conditions. Not sure what the cause is.
- [ ] Ubuntu: scroll bars are way too thin. Seen in Firefox running on Ubuntu. Setting `body.desktop.div scrollbar-width: unset` (or not setting it) solves on Ubuntu. 
- [ ] Macos/Ubuntu Firefox (only): Type icon in Inspector header is misaligned:
- [ ] Macos/Ubuntu Firefox (only): Number input spinners take up too much space, infringe on the input area and look bad. 

<img width="204" alt="image" src="https://user-images.githubusercontent.com/1056412/153241300-1814551d-33a0-4ffb-a0e5-631d1b1c42a5.png">
<img width="414" alt="image" src="https://user-images.githubusercontent.com/1056412/153242126-81252d99-e6f6-4fae-9777-b6dfec754236.png">

**Comment by charlesh88** [comment]
WIP in local branch `misc-ui-4843`.

**Comment by akhenry** [comment]
Can we file a PR for the WIP here an file followups for remaining work? The cost of maintaining long running branches increases exponentially over time as they drift further from master.

**Comment by charlesh88** [comment]
@akhenry Yes. Some of this may have been fixed already, as a first pass I'll double-check the items here against latest master.

**Comment by charlesh88** [comment]
### Testing Notes
#### Sub-object editing and more apparent Edit mode
1. Make or edit a Display Layout that contains a domain object and a drawing object.
2. While in Edit mode, verify that the main view header appears like this:

<img width="532" alt="image" src="https://user-images.githubusercontent.com/1056412/196962578-50dc2754-0d28-409a-bb80-fad39730f0ee.png">

3. Put that layout within another Display Layout. 
4. While in Edit mode, click the domain object within the nested layout.
5. Verify that it appears as selected:

<img width="601" alt="image" src="https://user-images.githubusercontent.com/1056412/196961748-e5ec5c5c-47b8-4447-b292-a1ffaeab7b0a.png">

6. Repeat steps 3 and 4 with the drawing object.
#### Edit Lock
1. In Browse mode, hover over the edit lock button in its unlocked state in the main view:

<img width="202" alt="image" src="https://user-images.githubusercontent.com/1056412/196963093-5aea8693-9db8-4566-b49d-592e004a1c85.png">

2. Verify that the title attribute that displays reads as shown above.
3. Click the lock to toggle it to locked, repeat step 1 and verify the title is the same.
